frontend/src/App.tsx

```diff
@@ -13,16 +36,69 @@ function MainPage() {
 }
 
 export default function App() {
+    const classes = useStyles();
     return (
         <Router>
-            <Switch>
-                <Route path='/Tokens'>
-                    <TokenPage />
-                </Route>
-                <Route path='/'>
-                    <MainPage />
-                </Route>
-            </Switch>
+            <Grid
+                container
+                direction='row'
+                justify='space-between'
+                alignItems='center'
+                className={classes.header}
+            >
+                <Typography variant='h5' className={classes.bannerText}>
+                    {'Hi, Jane'}
+                </Typography>
+                {/* Figure out sizes */}
+                <Grid item lg={2} sm={2} xs={2}>

```

Comment:
I think you can get away with only specifying the smallest breakpoint (xs) here and the larger values will default to that value unless specified but I'm having trouble finding the reference.

Reply:
That's a good point. I'll double check that